controls/radexpander/features/header-button-style.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Header Button Style
```

Comment:
I would name the article something like "Customizing Header Button". And I would consider moving it on the main level in the navigation tree or in a new folder named Appearance, instead of adding it to the Features folder.